core/services/directrequestocr/orm_test.go

```diff
@@ -1,41 +1,180 @@
 package directrequestocr_test
 
 import (
-	"fmt"
 	"testing"
 	"time"
 
-	"github.com/ethereum/go-ethereum/common"
 	"github.com/stretchr/testify/require"
 
+	"github.com/smartcontractkit/sqlx"
+
+	"github.com/smartcontractkit/chainlink/core/internal/testutils"
+	"github.com/smartcontractkit/chainlink/core/internal/testutils/pgtest"
+	"github.com/smartcontractkit/chainlink/core/logger"
 	"github.com/smartcontractkit/chainlink/core/services/directrequestocr"
 )
 
-func reqID(id uint) directrequestocr.RequestID {
-	byteArr := (*[32]byte)([]byte(fmt.Sprintf("%032d\n", id)))
-	return *byteArr
+type TestORM struct {
+	directrequestocr.ORM
+
+	db *sqlx.DB
+}
+
+func setupORM(t *testing.T) *TestORM {
+	t.Helper()
+
+	var (
+		db       = pgtest.NewSqlxDB(t)
+		lggr     = logger.TestLogger(t)
+		contract = testutils.NewAddress()
+		orm      = directrequestocr.NewORM(db, lggr, pgtest.NewQConfig(true), contract)
+	)
+
+	return &TestORM{ORM: orm, db: db}
+}
+
+func newRequestID() directrequestocr.RequestID {
+	return testutils.Random32Byte()
+}
+
+func TestORM_CreateRequestsAndFindByID(t *testing.T) {
+	t.Parallel()
+
+	orm := setupORM(t)
+	id1, id2 := newRequestID(), newRequestID()
```

Comment:
You can consider tiny helpers like these to make tests shorter and more readable:
https://github.com/smartcontractkit/chainlink/blob/develop/core/services/ocr2/plugins/directrequestocr/reporting_test.go#L58-L74

Reply:
I was thinking about this, but the tests are not much complicated. If you want, I can improve this though, no problems.

Reply:
Done.